os/booting-on-vmware.md

```diff
@@ -45,11 +63,20 @@ The last step uploads the files to your ESXi datastore and registers your VM.
 You can now tweak the VM settings, like memory and virtual cores, then power it
 on. These instructions were tested to deploy to an ESXi 5.5 host.
 
+### Booting with VMware Workstation 12
+
+Run VMware Workstation GUI:
```

Comment:
Would screenshots help here? My familiarity with these tools is lacking so if people can easily understand this I'm fine with no screenshots.